get-started/07_multi_container.md

```diff
@@ -59,7 +59,7 @@ For now, we will create the network first and attach the MySQL container at star
     ```console
     $ docker run -d \
         --network todo-app --network-alias mysql \
-        --platform "linux/amd64" \
+        --platform "linux/arm64" \
```

Comment:
@ankurgakhar Thanks for the update. MySQL 5.7 is available only for amd64. Therefore, the example here intentionally mentions amd64.

Reply:
Closing the ticket for the reason mentioned above. Feel free to comment after I close.